### DISCUSSION TITLE
全屏的音乐界面是怎么配置的呢？

### DISCUSSION THREAD

**Discussion by EternalDung** [liuzhihang/halo-theme-hao#688] [Q&A]
### 是什么版本出现了此问题？

1.4.8

### 在线站点地址

https://blog.sunguoqi.com/music/?id=4863092223&server=netease

### 发生了什么？

这是我看别人的博客可以自定义全屏的音乐界面，但是这应该不算自定义页面吧，配置里也只找到了音乐小部件

**Comment by carolcoral** [comment]
<img width="685" alt="image" src="https://github.com/liuzhihang/halo-theme-hao/assets/8952687/eeb74cba-a4d8-431b-8b57-bce9138a061b">
